site/app/models/Config.php

```diff
@@ -224,6 +226,10 @@ class Config extends AbstractModel {
     protected $queue_contact_info;
     /** @prop @var string */
     protected $queue_message;
+    /** @prop @var string */
+    protected $submitty_install_dir;
```

Comment:
should be `$submitty_install_path` to match the other path properties (e.g. `submitty_path`, `course_path`)